### PR TITLE
Update 07.pathogenesis.md

### PR DESCRIPTION
**Pull request by lubianat**
Adding small changes highlighted previously via hypothes.is

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by agitter — COMMENTED** [review]
These updates generally look good to me.  I'm not going to approve because I'd like @rando2 to have a chance to review, especially the immunology-related changes.  I also recall early discussion about SARS-CoV versus SARS-CoV-1 terminology.

**Comment by rando2** [comment]
Thank you @agitter! I am slowly working my way through so should be able to merge it in shortly :)

File: content/07.pathogenesis.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In this way, both biomedicine and genomics are important pieces of the puzzle of
 
 ### Fundamental Viral Pathogenesis
 
-As discussed above, the virus clusters with known coronaviruses (order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*).
+As discussed above, the virus genomic sequence clusters with known coronaviruses (order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*).
```
Comment (rando2) [inline content/07.pathogenesis.md:13]:
```suggestion
As discussed above, the viral genomic sequence clusters with known coronaviruses (order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*).
```

File: content/07.pathogenesis.md [hunk 2]
```diff
@@ -152,8 +152,8 @@ Acute phase response to an infection can also cause damage to the capillary endo
...
 The heightened inflammatory response in the lungs can also serve as a source for systemic inflammation, or sepsis, which can lead to multi-organ failure [@doi:10.1128/MMBR.05015-11].
 The shift from local to systemic inflammation is a phenomenon often referred to broadly as a cytokine storm [@doi:10.1128/MMBR.05015-11] or, more precisely, as cytokine release syndrome (CRS) [@doi:10.1186/s40425-018-0343-9].
-Sepsis is a known possible complication of pneumonia, and in an analysis of over 1400 US pneumonia patients, IL-6, tumor necrosis factor (TNF), and IL-1&beta; were found to be elevated at intake in patients who developed severe sepsis and/or ultimately deceased [@doi:10.1001/archinte.167.15.1655].
-IL-6 and TNF are pro-inflammatory cytokines, while IL-1&beta; is anti-inflammatory [@doi:10.1001/archinte.167.15.1655].
+Sepsis is a known possible complication of pneumonia, and in an analysis of over 1400 US pneumonia patients, IL-6, tumor necrosis factor (TNF), and IL-10; were found to be elevated at intake in patients who developed severe sepsis and/or ultimately deceased [@doi:10.1001/archinte.167.15.1655].
```
Comment (rando2) [inline content/07.pathogenesis.md:155]:
```suggestion
Sepsis is a known possible complication of pneumonia, and in an analysis of over 1,400 US pneumonia patients, IL-6, tumor necrosis factor (TNF), and IL-10; were found to be elevated at intake in patients who developed severe sepsis and/or ultimately deceased [@doi:10.1001/archinte.167.15.1655].
```

**Review by rando2 — APPROVED** [review]
These also look great to me! I had just a couple of minor suggestions for phrasing.

**Pull request merged**